pkg/types/azure/platform.go

```diff
@@ -31,6 +31,9 @@ type Platform struct {
 	// ARMEndpoint is the endpoint for the Azure API when installing on Azure Stack.
 	ARMEndpoint string `json:"armEndpoint,omitempty"`
 
+	// RHCOSImage is the url for the RHCOS image to use, required for Azure Stack and cannot be used with Azure.
+	RHCOSImage string `json:"RHCOSImage,omitempty"`
```

Comment:
I forgot about the explain command. You can add it to the description of explain command by running
```
go generate ./pkg/types/installconfig.go
```